subnet-scripts/computehorde/install.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#no GPU
+
+if [ -f .env ]; then
+    export $(cat .env | grep -v '#' | awk '/=/ {print $1}')
+fi
+
+if [ -d ~/.bittensor/subnets/computehorde/ ]; then
+    cd ~/.bittensor/subnets/computehorde/
+else
+    echo "Directory ~/.bittensor/subnets/computehorde/ does not exist. Exiting."
+    exit 1
+fi
```

Comment:
```suggestion
#!/bin/bash -eu
#no GPU

export $(awk '!/^#/ && /=/ {print}' .env)
cd ~/.bittensor/subnets/computehorde/
```
rather than checking the result of every single bash statement you can use -e to make it crash on non-successful execution (ofcourse `test`, `[` and pipelines are exempt)